_posts/development-guide/2019-09-05-xr-iframe-api.md

```diff
@@ -0,0 +1,60 @@
+---
+title: xr-iframe API
+description: How to create and manipulate reality tabs.
+categories:
+  - development-guide
+type: Document
+set: development-guide
+set_order: 21
+---
+
+This page will cover how to create and manipulate reality tabs (`xr-iframe`).
+
+## Top-level creation
+```js
+  // Import exokit-web
+  import('https://web.exokit.org/src/index.js'))
+  .then(() => {
+    // Create xr-iframe, define src attribute, and append/place it wherever you want as if it was a normal canvas
+    const xrIframe = document.createElement('xr-iframe');
```

Comment:
Maybe we should call this an `xr-scene`? It is different from the `xr-iframe` in usage.

Reply:
That makes sense conceptually, since it is possible to render [multiple scenes in something like threejs](https://jsfiddle.net/mmalex/sqg0d8vx/).

Reply:
Unlike three though, they would basically be completely separate.